virttest/utils_test/qemu/__init__.py

```diff
@@ -505,6 +506,23 @@ def get_guest_free_mem(cls, vm):
         finally:
             session.close()
 
+    @classmethod
+    def get_guest_crashkernel_size(cls, vm):
+        """
+        Guest OS reported crashkernel size in MB.
+
+        :param vm: VM Object
+        :return: crashkernel report by guest OS in MB
+        """
+        timeout = float(vm.params.get("login_timeout", 600))
+        try:
+            session = vm.wait_for_login(timeout=timeout)
```

Comment:
It should be out of the `try` block. When creating `session` fails, this will raise an exception, and in the `finally` part, will try to close `session`, which getting the second exception.

Reply:
thanks a lot, has been corrected.